README.md

```diff
@@ -296,12 +296,12 @@ This file may be good for experienced developers who:
     </td>
   </tr>
   <tr>
-    <!-- Technology --><td>Load balancer</td>
+    <!-- Technology --><td>L7 load balancer</td>
     <!-- Yandex internal --><td>
       <a href="https://github.com/yandex/balancer">Balancer</a> (Archived)
     </td>
     <!-- Analogs --><td>
-      <a href="https://github.com/yandex/balancer">Balancer</a>(Open source by Yandex)
+      <a href="https://github.com/envoyproxy/envoy">Envoy</a>
```

Comment:
I try to add links to open source Yandex projects in both columns. Balancer is quite tricky topic because its repository is archived. However I propose to keep link to its repository in both columns.

Reply:
I understand. Since you intend the list to be practical I suggest keeping the envoy link. The current production has much more in common with envoy than its own 9 year old version.

Reply:
Sounds reasonable. In that case LGTM :+1: